src/features/components/molecules/timeline-item/style.module.scss

```diff
@@ -0,0 +1,23 @@
+.timeline_item {
+    max-inline-size: 74.4rem;
+    position: relative;
+    &:before {
+        position: absolute;
+        content: "";
+        left: 10px;
+        border-inline-start: 2px dashed #FF444F;
```

Comment:
Can add the `$color-brand` instead of `#FF444F`

Reply:
Done..